### ISSUE TITLE
Unable to use Rancher CLI with SAML auth 

### ISSUE THREAD

**Issue by mrajashree** [rancher/rancher#16530]
Rancher/server:v1.6.14 and 1.6.24-rc1

Steps to repro:
1. Enable SAML auth
2. Create access and secret key
3. Use that with rancher cli to list environments

Expected
All envs should be listed

Actual
Only environments that the user creates are returned, envs that user is added to aren't listed

**Comment by alena1108** [comment]
The fix is available in `rancher/server:v1.6.26-rc3`

**Comment by sangeethah** [comment]
Tested with rancher server version - v1.6.26-rc4

Able to use Rancher CLI to list all environment the user has access to.

Setup - SAML (Ping Federator) auth enabled setup.
As admin user , Create following environments and give access to another user say "test1":
hello1 - Add "test1" as `member` to this env
hello2 - Add "test1" as `owner` to this env
Default - Add "test1" as `member` to this env

Using the access/secret keys of "test1" user , able to list all the environment user have access to:

```
./rancher --url http://<ip>:8080 --access-key <accesskey> --secret-key <secret-key> env ls 
ID        NAME      ORCHESTRATION   STATE     CREATED
1a17      hello1    cattle          active    2019-01-24T19:43:40Z
1a18      hello2    cattle          active    2019-01-24T19:43:52Z
1a5       Default   cattle          active    2019-01-24T19:11:39Z
``` 

Also able to list services/containers in the environment user has access to using `ps` command.

Remove user "test1" from environment "hello2".

When accessing environment list using rancher cli , "hello2" environment is not listed anymore as expected.
```
./rancher --url http://<ip>:8080 --access-key <accesskey> --secret-key <secret-key> env ls 
ID        NAME      ORCHESTRATION   STATE     CREATED
1a18      hello2    cattle          active    2019-01-24T19:43:52Z
1a5       Default   cattle          active    2019-01-24T19:11:39Z
```


Rancher CLI is not able to list environments that are given access to a user using group association. This is a know limitation at this point.
